driver/ssh/testdata/test_key

```diff
@@ -0,0 +1,39 @@
+-----BEGIN OPENSSH PRIVATE KEY-----
```

Comment:
We might want to generate those keys on the fly for the tests. It might create some noise in our security alerts for leaked passwords.